src/Service/SearchService.php

```diff
@@ -2,26 +2,51 @@
 
 namespace EMS\CoreBundle\Service;
 
+use Elastica\Query\BoolQuery;
+use EMS\CommonBundle\Service\ElasticaService;
+use EMS\CoreBundle\Entity\Environment;
 use EMS\CoreBundle\Entity\Form\Search;
-use EMS\CoreBundle\Entity\Form\SearchFilter;
+use EMS\CommonBundle\Search\Search as CommonSearch;
```

Comment:
```suggestion
use EMS\CommonBundle\Search\Search;
```
Call the other search import EntitySearch

Reply:
se also use EMS\CoreBundle\Entity\Form\Search;